README.md

```diff
@@ -1,17 +1,18 @@
-# Thrift Utilities for Elixir
+# A Thrift implementation for Elixir
 
 [![Build Status](https://travis-ci.org/pinterest/elixir-thrift.svg?branch=master)](https://travis-ci.org/pinterest/elixir-thrift)
 [![Coverage Status](https://coveralls.io/repos/pinterest/elixir-thrift/badge.svg?branch=master&service=github)](https://coveralls.io/github/pinterest/elixir-thrift?branch=master)
 
-This package contains a handful of useful utilities for working with
-[Thrift](https://thrift.apache.org/) in Elixir.
+This package contains a complete implementation of [Thrift](https://thrift.apache.org/) for Elixir. It includes a Thrift IDL parser, a code generator, a binary framed client and a binary framed server.
+
+The serialization and deserialization code that is generated by this project is highly optimized and is between **20 and 40 times faster** than the code generated by the Apache Erlang implementation. 
```

Comment:
Need to provide benchmark results, or preferably directions how to reproduce the benchmark.